elk/utils/data_utils.py

```diff
@@ -132,11 +132,12 @@ def binarize(template: Template, label: int, new_label: int, rng: Random) -> Tem
     true = answer_choices[label]
     false = rng.choice([c for c in answer_choices if c != true])
 
-    assert new_label in (0, 1)
+    # What order are we going to present the answer choices in?
+    new_label = rng.choice([0, 1])
```

Comment:
This won't work because all variants for a given example must have the same new_label

Reply:
sigh

Reply:
Technically we _could_ give all variants different labels but let's not do that

Reply:
I reluctantly just reverted this— really we need to destroy this and replace it with something actually correct